reflex/utils/hosting.py

```diff
@@ -19,32 +19,55 @@
 
 from reflex import constants
 from reflex.base import Base
+from reflex.config import get_config
 from reflex.utils import console
 
+config = get_config()
+# Endpoint to create or update a deployment
+POST_DEPLOYMENTS_ENDPOINT = f"{config.cp_backend_url}/deployments"
```

Comment:
this sort of thing makes me a little nervous because `get_config()` can return a different object during the lifetime of the process, however once this module is imported, the constants are stuck with whatever values were provided in the config at import time.

i don't think it's a problem now, but if we wanted to expose `cp_backend_url` as a command line flag (for example), then we'd have some more complex refactoring in this module to deal with the fact that the config when imported isn't actually the config we wanted to use.